Kruskal_Algorithm/Kruskal_Algorithm.js

```diff
@@ -0,0 +1,64 @@
+// Javascript Implementation of Kruskals Algorithm
+
+function kruskal() {
+    // Initialize graph that'll contain the MST
+    const MST = new Graph();
+    this.nodes.forEach(node => MST.addNode(node));
+    if (this.nodes.length === 0) {
+       return MST;
+    }
+ 
+    // Create a Priority Queue
+    edgeQueue = new PriorityQueue(this.nodes.length * this.nodes.length);
+ 
+    // Add all edges to the Queue:
+    for (let node in this.edges) {
+       this.edges[node].forEach(edge => {
+          edgeQueue.enqueue([node, edge.node], edge.weight);
+       });
+    }
+ 
+    let uf = new UnionFind(this.nodes);
+ 
+    // Loop until either we explore all nodes or queue is empty
+    while (!edgeQueue.isEmpty()) {
+       // Get the edge data using destructuring
+       let nextEdge = edgeQueue.dequeue();
+       let nodes = nextEdge.data;
+       let weight = nextEdge.priority;
+ 
+       if (!uf.connected(nodes[0], nodes[1])) {
+          MST.addEdge(nodes[0], nodes[1], weight);
+          uf.union(nodes[0], nodes[1]);
+       }
+    }
+    return MST;
+ }
+
+///Sample Input
+// let g = new Graph();
```

Comment:
Hey! I am not sure how to run this code. Can you please make it a little easier. In JS, you can write the input in the code itself.

Reply:
Okay I will just do that

Reply:
> Hey! I am not sure how to run this code. Can you please make it a little easier. In JS, you can write the input in the code itself.

I have made the changes, kindly check